<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMarkerDialog</class>
 <widget class="QDialog" name="EditMarkerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Marker</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="markerXLabel">
       <property name="text">
        <string>Marker X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="markerXEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="markerYLabel">
       <property name="text">
        <string>Marker Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="markerYEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="markerSizeXLabel">
       <property name="text">
        <string>Marker Size X:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="markerSizeXEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="markerSizeYLabel">
       <property name="text">
        <string>Marker Size Y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="markerSizeYEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="markerThicknessLabel">
       <property name="text">
        <string>Marker Thickness:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="markerThicknessEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="markerColorLabel">
       <property name="text">
        <string>Marker Color:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="markerColorSelectorButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="roiWidthLabel">
       <property name="text">
        <string>Marker ROI width:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="roiWidthEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="roiHeightLabel">
       <property name="text">
        <string>Marker ROI height:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="roiHeightEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markerColorSelectorButton</sender>
   <signal>clicked()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>showColorSelector()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roiWidthEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>autoFillRoiHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roiHeightEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>autoFillRoiWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roiWidthEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>autoFillRoiHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roiHeightEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>EditMarkerDialog</receiver>
   <slot>autoFillRoiWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showColorSelector()</slot>
  <slot>autoFillRoiHeight()</slot>
  <slot>autoFillRoiWidth()</slot>
 </slots>
</ui>
